code/build/template/include/[+inspector orxInspector.h]

```diff
@@ -27,6 +27,13 @@ orxBOOL                                     orxInspector_IsObjectRegistered(cons
 #define orxINSPECTOR_KZ_INPUT_SET           "orx:input:inspector"
 #define orxINSPECTOR_KZ_INPUT_INSPECT       "Inspect"
 
+#define orxINSPECTOR_KZ_TRIGGER             "Inspector"
+#define orxINSPECTOR_KZ_TRIGGER_REGISTER    "Register"
+#define orxINSPECTOR_KZ_TRIGGER_UNREGISTER  "Unregister"
+
+#define orxEVENT_TYPE_INSPECTOR             orxEVENT_TYPE_LAST_RESERVED

```

Comment:
I picked `orxEVENT_TYPE_LAST_RESERVED` here because it's currently very far away from potential conflicts with engine plugins' custom events. I'm happy to change this to something else now or later.

Reply:
We can add a new range for extensions. Otherwise it should technically be in the user range.

Reply:
https://github.com/orx/orx/pull/105/commits/fb5b0011265b7113c805c84ad14222c724c758b1 adds a potential implementation for a new event range for extensions and uses it.